IDE/STM32Cube/wolfSSL_conf.h

```diff
@@ -177,6 +177,11 @@
     #define HAL_CONSOLE_UART huart2
     #define NO_STM32_RNG
     #define WOLFSSL_GENSEED_FORTEST
+#elif defined(STM32G071xx)
+    #define WOLFSSL_STM32G0
+    #define HAL_CONSOLE_UART huart2
+    #define NO_STM32_RNG
+    #define WOLFSSL_GENSEED_FORTEST
```

Comment:
Wondering if we should add a note to the IDE/STM32Cube/README that tells people they should replace WOLFSSL_GENSEED_FORTEST this in production builds?

Reply:
There is a compiler warning that prints. This is setup here and the F0 because there is no RNG hardware.